settings.py

```diff
@@ -584,7 +584,9 @@
 # Keep messages in the events queue only for 2 hours
 CELERY_EVENT_QUEUE_TTL = 2 * 60 * 60
 
-CELERY_TASK_SERIALIZER = 'json'  # Default value in celery 4.x
+# Default serializer should be changed back to 'json' after
+# https://github.com/celery/celery/issues/6759 is fixed
+CELERY_TASK_SERIALIZER = 'pickle'  # default is 'json'. See corehq.apps.celery.shared_task
```

Comment:
Can you make this comment a bit clearer. Something like "this value in commcare hq code". I read this comment to say "celery defaults to json, but we are setting this value to default to pickle in commcare hq," which isn't true. We still default to json, and I think that should be much clearer without someone needing to figure out how the patch we made works.

Reply:
https://github.com/dimagi/commcare-hq/pull/31713/commits/ede3093687278c3119d0461e32a5b586513c3b24